cmd/ctr/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2018 The containerd Authors.
```

Comment:
note: once #647 is pushed the year is no longer necessary. The containerd guys don't want the year.

Reply:
Will review that PR later. :P Need to finish the integration first. :)